pyproject.toml

```diff
@@ -52,3 +52,10 @@ datasource_link = "redash_stmo.data_sources.link:extension"
 datasource_version = "redash_stmo.data_sources.version:extension"
 handler_queryresults = "redash_stmo.handlers.query_results:extension"
 queryrunner_presto = "redash_stmo.query_runner.presto:extension"
+
+[tool.flit.entrypoints."redash.bundles"]
+datasource_link = "redash_stmo.data_sources.link"
+datasource_version = "redash_stmo.data_sources.version"
```

Comment:
This is now needed with the design changes upstream, basically when we have hybrid (backend&client) extensions, the bundles of for the client part need to be specified separately from the backend code to simplify the guessing we're doing in the upstream code.